### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by sz10101**
**(charset.c)** Fix mangling of array formatting
**(init.c)** add statement validation and additional messages indicating the state of each key + value found or defaulted 
**(lp3000.c, pci_channel_win32.c)** casts to reduce compiler whining 
**(niu.c)** add initialization message confirming settings. 
**(tpmux.c)** Add port specification for tpmux and add initialization message. 
**(window_win32.c)** adjust thread priorities to help minimize "flashing" effect during repaint.  _(This is one of a few other attempts to clean up the faltering of the console display on windows.)_

### CODE REVIEW

File: tpmux.c [hunk 1]
```diff
@@ -316,16 +316,16 @@ void tpMuxInit(u8 eqNo, u8 unitNo, u8 channelNo, char *params)
 
     if (params != NULL)
         {
-        cp = strchr(params, ',');
-        if (cp != NULL) *cp = '\0';
+        //  Parse the TCP Port Number
+        cp = strtok(params, ", ");
```
Comment (kej715) [inline tpmux.c:320]: This call to strtok is superfluous. I recommend deleting this line and the declaration of the cp variable earlier; strtol will stop parsing if it should encounter a delimiter (which was the original purpose of searching for ',' and replacing it with '\0').

File: init.c [hunk 2]
```diff
@@ -362,115 +409,172 @@ static void initCyber(char *config)
...
+        {
+            goodToken = FALSE;
+            for (curVal = sectVals; curVal->valName != NULL; curVal++)
+            {
+                if (strncasecmp(curVal->sectionName, "cyber", 5) == 0)
+                {
+                    if (strcasecmp(curVal->valName, token) == 0)
+                    {
+                        goodToken = TRUE;
+                        fprintf(stderr, "(init   ) Section [%s], relative line %05d, value '%s'(%s) (file '%s')\n",
```
Comment (kej715) [inline init.c:444]: What is magical about displaying so many leading 0's on relative line numbers? It is very unlikely that any section would have as many as 10000 lines. For example, the cyber.ini file supporting the install-from-scratch NOS 2.8.7 system is relatively elaborate, yet the entire thing has less than 200 lines. Its largest section has fewer than 45.

**Review by kej715 — CHANGES_REQUESTED** [review]
Please update the tpmux.c module as indicated in the comments, and consider changing the %05d format specifiers in init.c to, e.g., %03d.

**Comment by sz10101** [comment]
I'll make the adjustments tomorrow.

Thanks!
Steve

**Pull request merged**